media-sound/mpdscribble/files/mpdscribble-0.23-Don-t-install-AUTHORS-COPYING-NEWS-README.rst.patch

```diff
@@ -0,0 +1,29 @@
+From 0921e6816d390d219f4e7a4fe72d10a17d67e359 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Arsen=20Arsenovi=C4=87?= <arsen@aarsen.me>
+Date: Thu, 9 Sep 2021 14:19:01 +0200
+Subject: [PATCH] Don't install AUTHORS, COPYING, NEWS, README.rst
+
+We install these in ebuilds.
+---
+ meson.build | 5 -----
+ 1 file changed, 5 deletions(-)
+
+diff --git a/meson.build b/meson.build
+index b4cc736..28ade21 100644
+--- a/meson.build
++++ b/meson.build
+@@ -128,11 +128,6 @@ executable(
+   install: true
+ )
+ 
+-install_data(
+-  'AUTHORS', 'COPYING', 'NEWS', 'README.rst',
+-  install_dir: join_paths(get_option('datadir'), 'doc', meson.project_name()),
+-)
+-
+ subdir('systemd')
+ 
+ subdir('doc')
+-- 
+2.32.0
+
```

Comment:
I guess there's some sort of conflict with compressing these files properly?

Reply:
yes, a version with and without compression would be installed